app.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "collections",
+  "repository": "https://github.com/alphagov/collections",
+  "env": {
+    "GOVUK_APP_DOMAIN": {
+      "value": "www.gov.uk"
+    },
+    "GOVUK_ASSET_ROOT": {
+      "value": "https://govuk-collections.herokuapp.com"
```

Comment:
@tijmenb I've added a new commit for this: b9917422b71501db69cfb6287ac3453204add419